### PR TITLE
add-missing-vars-desc

### PR DESCRIPTION
**Pull request by masterwishx**
Adding some missing description for cmdvartab and nut-names.txt.

After @desertwitch (nut plugin for Unraid developer) kindly added new future (show description form cmdvartab on hover for nut variables) asked by me, found some of them missing description.

### CODE REVIEW

**Comment by masterwishx** [comment]
Found some description was missing , should be also all `device.x` added ?
Seems they present in : 
docs/nut-names.txt
```

[options="header"]
|====================================================================================
| Name                | Description                                | Example value
| device.model        | Device model                               | BladeUPS
| device.mfr          | Device manufacturer                        | Eaton
| device.serial       | Device serial number (opaque string)       | WS9643050926
| device.type         | Device type (ups, pdu, scd, psu, ats)      | ups
| device.description  | Device description (opaque string)         | Some ups
| device.contact      | Device administrator name (opaque string)  | John Doe
| device.location     | Device physical location (opaque string)   | 1st floor
| device.part         | Device part number (opaque string)         | 123456789
| device.macaddr      | Physical network address of the device     | 68:b5:99:f5:89:27
| device.uptime       | Device uptime in seconds                   | 1782
| device.count        | Total number of daisychained devices       | 1
|====================================================================================

```

File: docs/nut-names.txt [hunk 1]
```diff
@@ -534,6 +536,7 @@ battery: Any battery details
                                  to "Warning" state (percent)        | 50
 | battery.charger.status       | Status of the battery charger
                                  (see the note below)                | charging
+| battery.charger.type         | ABM charger type                    | ABM
```
Comment (jimklimov) [inline docs/nut-names.txt:539]: I think this description can be generalized to "Type of battery charger (e.g. ABM)" which can eventually grow to other values as drivers learn to supply them. Same for `cmdvartab` then.

**Review by jimklimov — APPROVED** [review]

**Comment by jimklimov** [comment]
Thanks, the other values would be good too while we are at it :)

**Comment by masterwishx** [comment]
I hope not missed some variables ....

**Comment by masterwishx** [comment]
not sure where the error

**Comment by masterwishx** [comment]
@jimklimov Thanks, can be merged if we are not missed something...

**Pull request merged**